src/styles/variables/bulma-variables.scss

```diff
@@ -2,17 +2,17 @@
 $family-sans-serif: #{$p-fallback-fonts};
 
 // Colors
-$primary: $main-color;
+$primary: $primary;
 
 // Navbar
 $navbar-item-color: #fff;
-$navbar-background-color: $main-color;
+$navbar-background-color: $primary;
 $navbar-item-hover-background-color: #fff;
```

Comment:
@gonzalo-plaza Maybe we can use the white variable here

Reply:
Okay!! I correct this before the meeting of this night